src/main/kotlin/com/neva/gradle/fork/config/properties/Property.kt

```diff
@@ -31,9 +31,27 @@ class Property(
   val invalid: Boolean
     get() = validate().hasErrors()
 
-  fun validate(): PropertyValidator = PropertyValidator(this).apply(definition.validator)
+  var enabled: Boolean = true
 
-  fun other(name: String): Property {
-    return others[name] ?: throw PropertyException("Property named '$name' is not defined.")
+  fun control() = definition.controller(this)
```

Comment:
Name controller is unclear to me. Not sure what this is all about.
Maybe controlAvailabilityOfOtherProperties?

Reply:
it is a hook for controlling field validation, enable state in case of other fields; for me it's okey ;> and yours is too specific